lti_tool/views.py

```diff
@@ -0,0 +1,309 @@
+from django.http import HttpResponse
+from django.views.decorators.clickjacking import xframe_options_exempt
+from django.views.decorators.csrf import csrf_exempt
+from django.shortcuts import redirect
+from django.contrib.auth.models import User
+from django.contrib.auth import login
+from course.views import InstanceView, ModuleView
```

Comment:
Group and order the imports into django imports, pylti1p3 imports and A+ imports.